recipes/pymt_gridmet/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pymt_gridmet" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pymt-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8ebbf29c00afba9d7ca58e174bdcc77284b865403a36f7e094615e4bae397941
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
+  run:
+    - python >=3.6
+    - {{ pin_compatible('numpy') }}
```

This is only needed if you're building against the numpy C API (in which case it can't be noarch):
```suggestion
  run:
    - python >=3.6
    - numpy
```

---

@chrisburr Ah, thank you for this explanation. I'll apply your suggested fix.